src/fmu/dataio/_utils.py

```diff
@@ -463,3 +463,35 @@ def glue_metadata_preprocessed(
     meta["tracklog"].extend(newmeta["tracklog"])
 
     return meta
+
+
+def get_geometry_ref(geometrypath: str | None, obj: Any) -> dict[str, str]:
+    """Get a reference to a geometry.
+
+    Read the metadata file for an already exported file, and returns info like this
+    for the data block:
+
+    data:
+      geometry:
+        name: somename
+        relative_path: some_relative/path/geometry.roff
+
+    This means that the geometry may be 'located' both on disk (relative path) and in
+    SUMO
```

Comment:
*Sumo, not SUMO :)

Reply:
Ref section 8, paragraph 24 of the Sumo brand guidelines (revision 6): "Sumo is a name, not an abbreviation."